Line/iOS/LINELoginDemo/Podfile

```diff
@@ -0,0 +1,11 @@
+# Uncomment this line to define a global platform for your project
+# platform :ios, '9.0'
+
+target 'LINELoginDemo' do
+  # Uncomment this line if you're using Swift or would like to use dynamic frameworks
+  # use_frameworks!
+
+  # Pods for LINELoginDemo
```

Comment:
Unfortunately LINE doesn't provide their SDK through CocoaPods. You have to create a LINE business account in order to download the SDK as zip file.